aspnetcore/mvc/controllers/actions.md

```diff
@@ -14,80 +14,94 @@ uid: mvc/controllers/actions
 
 ## Defining Actions
 
-Any public method on a controller type is an action. Parameters on actions are bound to request data and validated using [model binding](../models/model-binding.md).
+Any public method on a controller type is an action. Parameters on actions are bound to request data and are validated using [model binding](xref:mvc/models/model-binding).
 
->[!WARNING]
+> [!WARNING]
 > Action methods that accept parameters should verify the `ModelState.IsValid` property is true.
 
-Action methods should contain logic for mapping an incoming request to a business concern. Business concerns should typically be represented as services that your controller accesses through [dependency injection](dependency-injection.md). Actions then map the result of the business action to an application state.
-Actions can return anything, but frequently will return an instance of `IActionResult` (or `Task<IActionResult>` for async methods) that produces a response. The action method is responsible for choosing *what kind of response*; the action result *does the responding*.
+Action methods should contain logic for mapping an incoming request to a business concern. Business concerns should typically be represented as services that the controller accesses through [dependency injection](xref:mvc/controllers/dependency-injection). Actions then map the result of the business action to an application state.
+
+Actions can return anything, but frequently return an instance of `IActionResult` (or `Task<IActionResult>` for async methods) that produces a response. The action method is responsible for choosing *what kind of response*; the action result *does the responding*.
 
 ### Controller Helper Methods
 
-Although not required, most developers will want to have their controllers inherit from the base `Controller` class. Doing so provides controllers with access to many properties and helpful methods, including the following helper methods designed to assist in returning various responses:
+Although not required, most developers have their controllers inherit from the base `Controller` class. Doing so provides controllers with access to three categories of helper methods designed to return specific response types:
 
-**[View](../views/index.md)**
+#### 1. An empty response body
 
-Returns a view that uses a model to render HTML. Example: `return View(customer);`
+No `Content-Type` HTTP response header is included, since the response body lacks content to describe.
+
+There are two response types within this category: Redirect and HTTP Status Code. The Redirect response type differs from the HTTP Status Code type primarily in the addition of a `Location` HTTP response header.
+
+**Redirect**
```

Comment:
This structure seems to have too many levels for clarity -- this is essentially an H5 rendered as just bold text.  Could the h5s be a bulleted list instead?